swagger-to-har.js

```diff
@@ -29,13 +29,26 @@ var Instantiator = require('./schema-instantiator.js')
  * @return {Object}         HAR Request object
  */
 var createHar = function (swagger, path, method) {
+  return createHarWithQueryValues(swagger, path, method, {})
```

Comment:
As above, I think `createHar` should just have an optional 4th parameter, rather than having two signatures.